collections/Bodl/MS_Bodl_851.xml

```diff
@@ -37,124 +37,780 @@
+                           (1973), Plate LXVI.</decoNote>
+                     </decoDesc>
+                  </physDesc>
+                  <history>
+                     <origin>
+                        <origDate calendar="Gregorian" notBefore="1390" notAfter="1500">14th
+                           century, late, or 15th century, early (?), with 15th century
+                           additions</origDate>
+                        <origPlace><country key="place_7002445">England</country>, <settlement
+                              key="place_7010823">Ramsey</settlement>, <orgName key="org_121821230"
+                              >Benedictine Abbey</orgName></origPlace>
+                     </origin>
+                  </history>
+               </msPart>
+
+               <msPart xml:id="MS_Bodl_851-partI">
+                  <msIdentifier>
+                     <altIdentifier><idno>MS. Bodl. 851, codicological unit I (fols. 7-77) </idno></altIdentifier>
+                  </msIdentifier>
                   <msContents>
-                     <msItem n="1" xml:id="MS_Bodl_851-part1-item1">
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem n="1">
+                        <locus from="7r" to="77r">(fols. 7r-73v)</locus>
                         <author key="person_98031958">Walter Map</author>
-                        <title key="work_4963">De nugis curialium</title>
-                        <textLang mainLang="la">Latin</textLang>
+                        <title key="work_4963">De Nugis Curlialium</title>
+                        <rubric>In libro magistri Gauteri Mahap de nugis curialium <gap
+                              reason="editorial"/></rubric>
+                        <incipit>In tempore sum et de tempore loquor</incipit>
+                        <explicit><locus>(fol. 72v)</locus> qui sedet super tronum et iudicabit
+                           iusticiam</explicit>
+                        <rubric><locus>(fol. 72v)</locus> Distinctio prima libri <gap
+                              reason="editorial"/></rubric>
+                        <note>Table of books and distinciones</note>
+                        <explicit>Explicit distinctio quinta libri magistri gauteri mahap de nugis
+                           curialium</explicit>
+                        <bibl>Printed from this, the unique manuscript, James, rev. Brooke and
+                           Mynors 1983.</bibl>
                      </msItem>
-                     <msItem class="#versus" n="2" xml:id="MS_Bodl_851-part1-item2">
-                        <title key="work_13886" type="desc">Poems</title>
-                        <textLang mainLang="la">Latin</textLang>
+
+                     <msItem n="2">
+                        <locus from="73va" to="73vb">(fols. 73va-73vb)</locus>
+                        <title>Fall of Carthage</title>
```

Comment:
I would probably leave this out, since translations are not provided for other titles.